main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -45,14 +45,14 @@ object Lowering {
   private object Defs {
     lazy val Box: Symbol.DefnSym = Symbol.mkDefnSym("Boxable.box")
 
-    lazy val Solve: Symbol.DefnSym = Symbol.mkDefnSym("Fixpoint.solve")
-    lazy val Merge: Symbol.DefnSym = Symbol.mkDefnSym("Fixpoint.union")
-    lazy val Filter: Symbol.DefnSym = Symbol.mkDefnSym("Fixpoint.project")
-    lazy val Rename: Symbol.DefnSym = Symbol.mkDefnSym("Fixpoint.rename")
+    lazy val Solve: Symbol.DefnSym = Symbol.mkDefnSym("Fixpoint/Solver.solve")
```

Comment:
I probably missed this early. But technically these paths should be `Fixpoint.Solver.solve` ,right?

Reply:
I am surprised the old-style still works

Reply:
I created a ticket. This seems to be a separate issue: https://github.com/flix/flix/issues/6940